code/game/objects/items/stacks/cable.dm

```diff
@@ -168,6 +168,51 @@ var/global/list/datum/stack_recipe/cable_recipes = list ( \
 				getFromPool(/obj/item/stack/cable_coil, C.loc)
 				returnToPool(C)
 
+//I really wish that redundant shit wasn't needed, but snowflake code yeah
+/obj/item/stack/cable_coil/proc/catwalk_place(obj/structure/catwalk/Catwalk, mob/user)
```

Comment:
Why does this proc exist at all?
It has the same functionality as the turf proc, only with the addition of catwalk typing.
You could have had it so that hitting a catwalk with a cable simply returned the turf proc, but with the catwalk's turf.